andela/learning-map-front
# #149840109 Create Contribution Filter

#### What does this PR do?
Create contribution filter section on contribution page

#### Description of Task to be completed?
Create contribution filter

#### How should this be manually tested?
After cloning the repo
- Run yarn install
- Run yarn start
- Access the landing page via localhost:3000 and click on the LOGIN TO CONTRIBUTE button.
- Navigate to contributions page to see card

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#149840109](https://www.pivotaltracker.com/story/show/149840109)

#### Screenshots (if appropriate)
<img width="627" alt="screen shot 2017-08-07 at 12 23 03" src="https://user-images.githubusercontent.com/25595450/29021112-1b318336-7b6d-11e7-940e-9e615fef56b6.png">
On clicking search
<img width="780" alt="screen shot 2017-08-07 at 12 22 46" src="https://user-images.githubusercontent.com/25595450/29021113-1b31dc64-7b6d-11e7-8817-f505ea344bcf.png">




#### Questions:
N/A
